tests/unittest/pywbem/testmofs/test_instance.mof

```diff
@@ -5,7 +5,7 @@
 #pragma include("test_types.mof")
 
 
-// TODO: Uncomment this to work on issues with instance MOF
+// TODO: Uncomment this to work on issues with embedded instance MOF
```

Comment:
Not sure I understand this TODO. It is already uncommented. ??

Reply:
Actually the only thing I did was added the word embedded.  This is from a very old set of code (Bart days) and refers to the last two lines of property  definitions in the following mof which are to compile instances of  embedded instance properties.  

In retrospect, a quick review indicates that the particular MOF line indicated are actually broken.  They are trying to use an alias to substitute the mof in embedded... propeties when, in fact,  the alias inserts a path.  I removed the references in this test case that are broken